src/rapidhtml/tags.py

```diff
@@ -105,6 +110,10 @@ def render(self):
             if value is None:
                 value = "none"
 
+            if value is True:
+                ret_html += f"{key} "
+                continue
```

Comment:
I'm not certain about this change. Basically, it means if an attribute has the value of `True` it will be added without a value, IE

```py
H1(some_value=True)
```
```html
<h1 some-value></h1>
```
and if you wanted the value to actually be `"true"` you'd need to pass it as a string
```py
H1(some_value="true")
```
```html
<h1 some-value="true"></h1>
```
Unfortunately there are some attributes that evaluate to true just by existing. I'm not sure why the best way to do this from a Python perspective is

Reply:
Perhaps a list of known boolean attributes?

Reply:
I dig it